soccer/src/rj_vision_filter/ball/WorldBall.cpp

```diff
@@ -22,17 +21,14 @@ WorldBall::WorldBall(RJ::Time calcTime,
 
     // Below 1 would invert the ratio of scaling
     // Above 2 would just be super noisy
-    if (*ball_merger_power < 1 || *ball_merger_power > 2) {
-
-        std::cout
-             << "WARN: ball_merger_power should be between 1 and 2"
-             << std::endl;
+    if (PARAM_ball_merger_power < 1 || PARAM_ball_merger_power > 2) {
+        std::cout << "WARN: ball_merger_power should be between 1 and 2"
```

Comment:
Yeah.... this is kind of hard because this is like `ROS` independent code and so there isn't an easy way to call ROS related logging things. I think this needs a longer discussion on how to do this.